ScreenManager.lua

```diff
@@ -34,6 +34,29 @@ local stack;
 local screens;
 local change;
 
+local allcallbacks = {
```

Comment:
So basically remove the whole table and do the loop inside the function if no callbacks table is provided right? That sounds good, I don't think you wanna call `registerCallbacks` more than once anyway (since that means that you will register twice and end up with repeated events)